### PR TITLE
[SI-165] Remove womens cats from Cat Lite Cypress test

### PR DESCRIPTION
**Pull request by a6software**
Includes:

* SI-108
* SI-63
* SI-35
* SI-24

### CODE REVIEW

File: server/views/formPages/supervisor/review.html [hunk 1]
```diff
@@ -226,14 +229,16 @@ <h2 class="govuk-heading-m">Other Information</h2>
...
           value: "No",
           text: noLabel,
           conditional: {
             html:overriddenCategoryTextHtml
-          }
+          },
+          checked: data.supervisor.review.supervisorCategoryAppropriate === 'No'
         }
       ]
     }) }}
```
Comment (a6software) [inline server/views/formPages/supervisor/review.html:244]: this is a bug fix - [SI-63]

File: integration_tests/e2e/ratings/securityInput.cy.ts [hunk 2]
```diff
@@ -125,7 +125,7 @@ describe('Security Input', () => {
         })
       })
 
-      describe("it should record a 'yes' decision", () => {
+      describe.only("it should record a 'yes' decision", () => {
```
Comment (Stubbs) [inline integration_tests/e2e/ratings/securityInput.cy.ts]: Is this `.only()` intentional?

**Review by Stubbs — APPROVED** [review]
In terms of code itself it looks fine, just check that `.only` on one of the tests, feels like it might be blocking a bunch of tests from running. I don't know the system well enough to comment on what they're covering though, @garethmdavies maybe?

**Review by garethmdavies — APPROVED** [review]
LGTM

**Pull request merged**